easybuild/easyconfigs/c/CGAL/CGAL-4.8.1-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+name = 'CGAL'
+version = '4.8.1'
+gmpver = '6.0.0a'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+# note: source URL needs to be updated for a new version (checksums too)!
+source_urls = ['https://github.com/%(name)s/%(namelower)s/releases/download/releases%2F%(name)s-%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['1cbcb3f251822766ed3d744f923c782d']
+
+dependencies = [
+    ('Boost', '1.60.0'),
+    ('GMP', gmpver),
+    ('MPFR', '3.1.2', '-GMP-%s' % gmpver),
+]
```

Comment:
@boegel the libGLU dependency is sending me on a trip through dependency Hades. I'm seeing if I can sort it out so that this can be tidied away...